### ISSUE TITLE
Guard mode not work with turret 

### ISSUE THREAD

**Issue by yaus** [BahamutoD/BDArmory#51]
All weapon on turret would not fire with guard mode.

**Comment by enjoybeta** [comment]
Many reason could lead to this: maybe too far, try stay in 3.6km; also maybe enemy vessels do not have weapon manger and any turrent, which all could cause this

**Comment by yaus** [comment]
No! The enemy vessels have weapon manger and able to target and fire manually.

**Comment by enjoybeta** [comment]
maybe no ammo? try a ciws and a plane flying at low attitude

**Comment by yaus** [comment]
No, I should clarify that is missile turret. Not gun turret.

**Comment by enjoybeta** [comment]
It is true that ground missile are quite ineffective to planes at the moment. They can not track planes flying high or can not turnto aim quick enough. BTW, do you have an lock-on radar? Maybe try fire manually is more effective, but quite likely to miss a lot

**Comment by yaus** [comment]
Not inefficient. It is not able to lock and fire the missile in guard mode with missile turret. It is all OK with manually.

**Comment by boomchacle** [comment]
is the radar on?

**Comment by TangentFoxy** [comment]
@yaus What version of BD and what version of KSP? Remember, BD has not been updated since 1.1 and has less compatibility with newer versions.
